### PR TITLE
Padms/1839

### PR DESCRIPTION
**Pull request by padms**

### CODE REVIEW

**Review by millianapia — APPROVED** [review]
good purge

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
   // "forwardPorts": [3000],
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": "pnpm m i && (cd ./studio ; yarn install)",
+  "postCreateCommand": "pnpm m i && (cd ./sanityv3 ; yarn install)",
```
Comment (fernandolucchesi) [inline .devcontainer/devcontainer.json:61]: sanity v3 uses pnpm, so we don't need the && cd blabla yarn install

File: package.json [hunk 2]
```diff
@@ -6,15 +6,13 @@
   },
   "type": "module",
   "scripts": {
-    "setup-project": "pnpm m i && pnpm studio install",
+    "setup-project": "pnpm m i && pnpm sanityv3 install",
```
Comment (fernandolucchesi) [inline package.json:9]: pnpm m i will work for sanityv3. no need to run it twice

File: .github/workflows/PROD-studios-rollback.yaml
Comment (fernandolucchesi) [inline .github/workflows/PROD-studios-rollback.yaml:1]: line 44, it is looking for studio-global image. I believe we have a different image name for v3

Comment (fernandolucchesi) [inline .github/workflows/PROD-studios-rollback.yaml:1]: We should also have a rollback-secret job, similar to rollback global with secrets variables

Comment (fernandolucchesi) [inline .github/workflows/PROD-studios-rollback.yaml:1]: it is probably a good idea to use the environment as a GH action input, allowing dev, preprod or prod.

**Review by fernandolucchesi — CHANGES_REQUESTED** [review]
Left some comments, regarding the rollback changes, I think it is a better idea to work on it in a separate issue. Then we don't wait too long to get the other changes into production. (No need to revert code already modified, since it doesn't work anyways)

File: .github/workflows/PROD-studios-rollback.yaml
Comment (padms) [reply .github/workflows/PROD-studios-rollback.yaml:1]: No its studio-global only. Its the dev which is syudiov3-global-development

**Pull request merged**